crates/uv-interpreter/src/python_environment.rs

```diff
@@ -93,13 +93,12 @@ impl PythonEnvironment {
     /// In most cases, `purelib` and `platlib` will be the same, and so the iterator will contain
     /// a single element; however, in some distributions, they may be different.
     pub fn site_packages(&self) -> impl Iterator<Item = &Path> {
-        std::iter::once(self.interpreter.purelib()).chain(
-            if self.interpreter.purelib() == self.interpreter.platlib() {
-                None
-            } else {
-                Some(self.interpreter.platlib())
-            },
-        )
+        let mut site_packages = Vec::new();
+        site_packages.push(self.interpreter.purelib());
+        site_packages.push(self.interpreter.platlib());
+        site_packages.extend(self.interpreter.sys_path());
+        site_packages.dedup();
```

Comment:
note this will only remove consecutive dedups. also the function doc will need updating

Reply:
will do once i've sorted through other issues, not quite sure if this will be the final form.